docs/tutorials/hello-world.md

```diff
@@ -180,17 +160,14 @@ println!(
 
 ## Custom Database and Blockchain types
 
-We briefly mentioned before that for our example we used the `MemoryDatabase`, but that it could also be swapped for a different one: this is one example of the *modularity* of BDK. By default, some database
-types are implemented in the library, namely (as of now) the `MemoryDatabase` which only keeps data in RAM and the [sled][sled] database that can store data on a filesystem. But since the `Database` trait is public,
-users of the library can also implement different database types more suitable for their use-case.
+We briefly mentioned before that for our example we used the `MemoryDatabase`, but that it could also be swapped for a different one: this is one example of the *modularity* of BDK. By default, some database types are already implemented in the library, namely the `MemoryDatabase` which only keeps data in RAM, the [sled][sled] database that can store data on a filesystem, and the `SqliteDatabase`[rusqlite] that can store data into a SQLite database. But since the `Database` trait is public, users of the library can also implement different database types more suitable for their use-case.
```

Comment:
The `rusqlite` link is missing..

Reply:
The rusqlite link misbehaved for me. Don't have time to fix the link. So, I removed the link.